scripts/android/travisbuild.sh

```diff
@@ -6,6 +6,6 @@ set -e
 # by running the build wrapper again we should be able to test newer
 # versions of the dependencies even without updating the docker image
 # (but of course having the right things in place will save a ton of time)
-docker exec -e TRAVIS="$TRAVIS" -t android-builder subsurface/packaging/android/android-build-wrapper.sh
+docker exec -e TRAVIS="$TRAVIS" -t android-builder sh -c "ln -s /android/Qt . ; ln -s /android/android-ndk-r18b . ; ln -s /android/android-sdk-linux . ; subsurface/packaging/android/android-build-wrapper.sh"
```

Comment:
I actually wonder how you tested that :-)

Reply:
Well, my local build process is:
- setup: "sh scripts/android/before_install.sh"
- run: "docker start android-builder && sh scripts/android/travisbuild" ;-)

Reply:
Cool :-)